lib/cacheWrapper.js

```diff
@@ -1,76 +1,112 @@
 'use strict';
 
 var cacheWrapper = module.exports = {};
-
-var _ = require( 'lodash' );
-var Catbox = require( 'catbox' );
-var catboxMemory = require( 'catbox-memory' );
-var catboxRedis = require( 'catbox-redis' );
-var Q = require( 'q' );
-
+var _ = require('lodash');
+var Catbox = require('catbox');
+var catboxRedis = require('catbox-redis');
+var redisWrapper = require('./redisWrapper');
+var Q = require('q');
 var cacheClient;
 var startingClient = false;
 var queryQueue = [];
 var policies = [];
+var partition = '';
+
+cacheWrapper.initialise = function(serverConfig, cachePolicies) {
+  // no config, no cache...
+  if (_.isEmpty(serverConfig)) {
+    return Q.reject('no serverConfig passed');
+  }
+  if (_.isEmpty(cachePolicies)) {
+    return Q.reject('cachePolicies missing options(segment/expiresIn)');
+  }
+  partition = serverConfig.partition;
```

Comment:
Should we reject the promise if the partition is empty, cacheWrapper.delete will fail in this case.